changelog/unreleased/issue-323

```diff
@@ -0,0 +1,11 @@
+Enhancement: Add a command to copy a snapshot to another repository
+
+We've added a copy command, allowing you to copy snapshots from one
+repository to another.
+
+Note that this process will have to read (download) and write (upload) the
+entire snapshot(s) due to the different encryption keys used on the source
+and destination repository.
```

Comment:
Suggest (as per @aawsome) adding "Also, the transferred files are not re-chunked, which may break deduplication between files already stored in the destination repo and files copied there using this command."

Reply:
Is added.